db/schema.rb

```diff
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_01_25_162730) do
-
```

Comment:
It seeems that somehow, the schema.rb of main is corrupted (!). The latest migration is 20220810152049_add_ghost_hash_to_user.rb, so schema version should be 2022_08_10_152049 as it is in the PR.